src/rotator_library/providers/utilities/gemini_credential_manager.py

```
@@ -165,43 +165,69 @@ async def _load_persisted_tiers(
                 continue
-
             # Skip if already in cache
             if path in self.project_tier_cache:
                 continue
+            paths_to_load.append(path)
+
+        if not paths_to_load:
+            return loaded
 
+        # Read all files in parallel
+        def _read_credential_file(path: str):
```

Ruff (0.15.4)

[warning] Missing return type annotation for private function `_read_credential_file`

(ANN202)